css/deeplinking.scss

```diff
@@ -0,0 +1,73 @@
+.deeplinking {
+    background-color: #fff;
+    width: 100%;
+    height: 100%;
+    .header {
+        width: 100%;
+        height: 70px;
+        background-color: #f1f2f5;
+        .logo {
+            margin-top: 15px;
+            margin-left: 50px;
+            height: 40px;
+            float: left;
+        }
+        .appName {
+            float: left;
+            margin-top: 15px;
+            margin-left: 5px;
+            height: 40px;
+            font-size: 25px;
+        }
+    }
+    .content {
+        top: 110px;
+        bottom: 40px;
+        left: 0px;
+        right: 0px;
+        position: fixed;
+        .leftColumn {
+            float: left;
```

Comment:
You seriously don't need float for CSS layout. You can get by with things like inline-block and flex (and nowadays grid on modern browsers). float can cause unexpected box interactions and the general best practices nowadays is not to use it for layout.

Reply:
OK. Changed to flex.